### PR TITLE
move Logo component to common components

### PR DESCRIPTION
**Pull request by abbasali1994**
Ticket(s):
- closes [#1826](https://github.com/AngelProtocolFinance/angelprotocol-web-app/issues/1826)

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
- verify all previous logos (on Profile, in app header (AP logo), in "My Endowments")

### CODE REVIEW

**Comment by abbasali1994** [comment]
@misicnenad @ap-justin can this <Logo /> be used with Marketplace cards as well?

**Comment by misicnenad** [comment]
> @misicnenad @ap-justin can this be used with Marketplace cards as well?

@abbasali1994 yes it could, it could be renamed to `Image` to be used for general image loading/display.

**Review by misicnenad — CHANGES_REQUESTED** [review]
There are a bunch of other `Logo.tsx` files/components, all (most) of them should be using this common one.

File: src/components/WalletSuite/WalletSelectorOpener/Connector.tsx [hunk 1]
```diff
@@ -22,26 +21,10 @@ export default function Connector(props: Connection) {
       className="flex flex-col items-center justify-center gap-1 h-28 p-5 border border-gray-l2 rounded bg-white hover:bg-orange-l5 dark:bg-blue/50 hover:dark:bg-blue-d3 dark:border-none"
       onClick={handleConnect}
     >
-      <Logo logo={props.logo} />
+      <Image src={props.logo} className={"w-12 h-12"} />
```
Comment (misicnenad) [inline src/components/WalletSuite/WalletSelectorOpener/Connector.tsx:24]: nit: no need for surrounding curly brackets, `...={"..."}` can be just `...="..."`

**Review by misicnenad — CHANGES_REQUESTED** [review]
There are other Logo's that can be replaced with the newly created `Image`:
`src/pages/Profile/Logo.tsx`
`src/App/Header/Logo.tsx`
`src/components/APLogo.tsx`

Also, [LogoPlaceholder](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/components/LogoPlaceholder.tsx) is used only in `Image` and should be renamed to `ImagePlaceholder`.

**Comment by misicnenad** [comment]
> There are other Logo's that can be replaced with the newly created `Image`: `src/pages/Profile/Logo.tsx` `src/App/Header/Logo.tsx` `src/components/APLogo.tsx`
> 
> Also, [LogoPlaceholder](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/components/LogoPlaceholder.tsx) is used only in `Image` and should be renamed to `ImagePlaceholder`.

@SovereignAndrey @abbasali1994 @ap-justin addressed remaining change requests.

In light of changes introduced in this PR, it would make sense to replace all uses of plain `` tags with this new general `Image` component as it is more powerful (supports loading logic+UI & placeholder images). Created a draft task proposal for this https://github.com/orgs/AngelProtocolFinance/projects/2/views/1?filterQuery=components%2F&pane=issue&itemId=22174640

**Review by SovereignAndrey — APPROVED** [review]

**Pull request merged**